iis-wf/iis-wf-affmatching/src/main/java/eu/dnetlib/iis/wf/affmatching/match/AffOrgMatcherFactory.java

```diff
@@ -99,16 +102,21 @@ public static AffOrgMatcher createDocOrgRelationMatcher(JavaSparkContext sparkCo
      */
     public static ImmutableList<AffOrgMatchVoter> createDocOrgRelationMatcherVoters() {
 
-        FittingOrgWordsMatchVoter fitOrgWordsMatchVoter = new FittingOrgWordsMatchVoter(ImmutableList.of(','), 3, 0.8f, 0.9f);
-        fitOrgWordsMatchVoter.setMatchStrength(0.814f);
+        FittingOrgWordsMatchVoter fitOrgWordsMatchVoter = new FittingOrgWordsMatchVoter(ImmutableList.of(','), 2, 0.7f, 0.9f);
```

Comment:
Probably we should add `;` semicolon to charsToFilter. (see `OrganizationNameNormalizer`)